src/plugins/plugin.legend.js

```diff
@@ -237,26 +237,54 @@ var Legend = Element.extend({
 			minSize.height = me.maxHeight; // fill all the height
 		}
 
+		var getLineWidths = function(boxWidth, textLines) {
```

Comment:
I would change this to `getMaxLineWidth`. Both places where you call this you loop through the result to find the largest. I would do that here to reduce the code duplication

Reply:
I've resolved this and the next review, and pushed, but the commit does not appear in the PR for some reason...

Reply:
Commit 740f4b0

Reply:
Force pushed to resolve